src/launcher.rs

```diff
@@ -11,12 +16,17 @@ use semver::Version;
 use serde::{Deserialize, Serialize};
 #[cfg(test)]
 use tempfile::TempDir;
-use tracing::{debug, info, warn};
+use tracing::{debug, error, info, warn};
 
 use crate::utils::{self, NodeExitCode};
 
 /// The name of the file for the on-disk record of the node-launcher's current state.
 const STATE_FILE_NAME: &str = "casper-node-launcher-state.toml";
+/// The path of the node-launcher shutdown script.
+#[cfg(not(test))]
+const SHUTDOWN_SCRIPT_PATH: &str = "/etc/casper/casper_shutdown_script";
+#[cfg(test)]
+const SHUTDOWN_SCRIPT_PATH: &str = "/tmp/test_casper_shutdown_script";
```

Comment:
Please consider leveraging the `TEMP_DIR` we already have defined [here](https://github.com/casper-network/casper-node-launcher/blob/main/src/launcher.rs#L45-L49). This applies to the "script output" as well.

Reply:
We can't use `TEMP_DIR` inside this constant because it is a `static`. Unfortunately, I didn't find any way out of this so far, but I will use `TEMP_DIR` for joining the output path.